bridge/discord/discord.go

```diff
@@ -208,7 +208,7 @@ func (b *Bdiscord) Send(msg config.Message) (string, error) {
 	b.channelsMutex.RUnlock()
 
 	// Use webhook to send the message
-	if wID != "" {
+	if wID != "" && msg.ID == "" {
```

Comment:
The permission error debug message (when editing) can be suppressed if I add a similar guard to the `// skip events` check below.